upgrading/topics/rhsso/changes-72.adoc

```diff
@@ -48,11 +48,11 @@ described in <<_compatibility_with_older_adapters>>. There is the `Exclude Sessi
 which can be turned on to prevent adding the `session_state` parameter to the Authentication Response.
 
 === Microsoft Identity Provider updated to use the Microsoft Graph API
-
-The Microsoft Identity Provider implementation in {project_name} up to version 7.2.4 relies on the Live SDK
```

Comment:
I'd remain version specification in changes-72.adoc but remove those in changes-73.adoc, because the versions make sense in 7.2 changes but not much sense in 7.3 changes. Makes sense?

Reply:
That makes a lot of sense. New commit en route.